workbench-core/base/src/interfaces/queryParameterFilter.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface QueryParameterFilter<T> {
+  eq?: T;
+  lt?: T;
+  lte?: T;
+  gt?: T;
+  gte?: T;
+  between?: { value1: T; value2: T };
```

Comment:
maybe `start` and `end` to indicate a range?

Reply:
changing the values here implies we need to do a parse of fields when doing the marshall to the object, so I felt this was more convenient